torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -394,24 +461,42 @@ void unpackQuantizedWeightsHelper(
         input_val->type()->str());
```

Comment:
Line 448 refers to caffe2 but is not inside a caffe2 if block. Please update comment.